### ISSUE TITLE
make a new release

### ISSUE THREAD

**Issue by laduke** [zerotier/go-ztidentity#4]
I don't think I have the credentials to do that.

**Comment by glimberg** [comment]
Think it's just adding a new tag?

**Comment by laduke** [comment]
oh sorry, i mean to go pkgs https://pkg.go.dev/github.com/zerotier/go-ztidentity

https://deps.dev/go/github.com%2Fzerotier%2Fgo-ztidentity/v1.0.0/dependents

**Comment by glimberg** [comment]
Yeah.  I think all you have to do is add a new tag & pkgs will pick it up as people use it
